pkg/autohealing/cloudprovider/openstack/provider.go

```diff
@@ -275,6 +330,8 @@ func (provider OpenStackCloudProvider) Repair(nodes []healthcheck.NodeInfo) erro
 		masters = nodes
 	}
 
+	firstTimeRebootNodes := make(map[string]healthcheck.NodeInfo)
```

Comment:
`firstTimeRebootNodes` is initialized as an empty map every time when `Repair()` is called, that's not what we expected.

Reply:
It's expected that the variable `firstTimeRebootNodes` is only available in the Repair function because all the broken nodes information are stored in variable `unHealthyNodes`. Maybe we should get a better name, the variable is used to temporarily storing broken nodes that have been processed by the function `firstTimeRepair`.